src/pages/storage/forms/StoragePoolFormZFS.tsx

```diff
@@ -0,0 +1,43 @@
+import { FormikProps } from "formik";
+import { FC } from "react";
+import { StoragePoolFormValues } from "./StoragePoolForm";
+import { getConfigurationRow } from "components/ConfigurationRow";
+import { Input, Select } from "@canonical/react-components";
+import { optionTrueFalse } from "util/instanceOptions";
+import ScrollableConfigurationTable from "components/forms/ScrollableConfigurationTable";
+
+interface Props {
+  formik: FormikProps<StoragePoolFormValues>;
+}
+
+const StoragePoolFormZFS: FC<Props> = ({ formik }) => {
+  return (
+    <ScrollableConfigurationTable
+      rows={[
+        getConfigurationRow({
+          formik,
+          label: "ZFS pool name",
+          name: "zfs_pool_name",
+          defaultValue: "",
+          children: <Input type="text" placeholder="Enter ZFS pool name" />,
+        }),
+        getConfigurationRow({
+          formik,
+          label: "Clone copy",
+          name: "zfs_clone_copy",
+          defaultValue: "",
+          children: <Select options={optionTrueFalse} />,
+        }),
+        getConfigurationRow({
+          formik,
+          label: "Export",
+          name: "zfs_export",
```

Comment:
In a clustered backend I got this error on creating a zfs pool with this option set to false:
Config key `zfs.export" may not be used as member-specific key`